robots:
  a_iiwa:
    default_gripper: a_iiwa_gripper
  b_iiwa:
    default_gripper: b_iiwa_gripper

grippers:
  a_iiwa_gripper:
    type:          RobotiqGripper
    controller_ns: a_iiwa_gripper_controller
    max_effort:    5.0
    velocity:      0.1
  b_iiwa_gripper:
    type:          SuctionGripper
    controller_ns: b_iiwa_gripper_controller
